common/app/views/fragments/collections/popular.scala.html

```diff
@@ -35,7 +35,14 @@
                     @section.trails.zipWithRowInfo.map{ case (trail, info) =>
                     <li class="headline-list__item headline-column__item headline-column--tablet__item headline-column--desktop__item @toneClass(trail)--most-popular">
                         <div class="headline-list__link" data-link-name="@info.rowNum | text">
-                            <p class="headline-list__count">@info.rowNum</p>
+                            @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)){
+                                <p class="headline-list__count">@info.rowNum</p>
+                            }
+                            @if(experiments.ActiveExperiments.isParticipating(experiments.Garnett)){
+                                <span class="most-popular__number-@info.rowNum">
+                                    @fragments.inlineSvg(s"number-${info.rowNum}", "numbers", List("most-popular__number-svg svg-number"))
```

Comment:
Why add these classes, if you are not using them?`most-popular__number-svg svg-number`